rts/at/undertow/src/main/resources/META-INF/beans.xml

```diff
@@ -0,0 +1,6 @@
+<beans xmlns="https://jakarta.ee/xml/ns/jakartaee"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
maven shade plugin (https://github.com/jbosstm/quickstart/blob/main/rts/at/undertow/pom.xml#L101) should be upgraded, I suggest to do it with a plugin Management in Narayana bom. My idea is that this old plugin generates old beans.xml, while we want to use a newer one with bean-discovery-mode = annotated. Please note this change is needed at the moment, otherwise the test fails.